janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/formats/util/HadoopRecordReader.java

```diff
@@ -65,7 +65,7 @@ public void initialize(final InputSplit inputSplit, final TaskAttemptContext tas
     public boolean nextKeyValue() throws IOException, InterruptedException {
         while (reader.nextKeyValue()) {
             // TODO janusgraph05 integration -- the duplicate() call may be unnecessary
-            final TinkerVertex maybeNullTinkerVertex =
+            final Vertex maybeNullTinkerVertex =
```

Comment:
nitpick: we should rename this variable as it is now not a "TinkerVertex".

Reply:
ack